packages/pirania-app/files/usr/libexec/rpcd/pirania-app

```diff
@@ -43,27 +72,15 @@ local function get_clients(msg)
 end
 
 local function read_content(msg)
-    local contents = ""
-    local myTable = {}
-    local file = io.open( path, "r" )
-
-    if file then
-        -- read all contents of file into a string
-        local contents = file:read( "*a" )
-        myTable = json.decode(contents)
-        io.close( file )
-    end
-    printJson(myTable)
+    local result = {}
```

Comment:
sounds like there is a generalization of reading a printing a JSON file (here and in governance you do the same)

Reply:
:+1: